controllers/helper.go

```diff
@@ -397,3 +398,21 @@ func updateDaemonsetNodeSelector(obj *uns.Unstructured, nodeSelector map[string]
 	}
 	return nil
 }
+
+func isPrometheusOperatorInstalled(ctx context.Context, client k8sclient.Reader) bool {
+	u := &uns.UnstructuredList{}
+	u.SetGroupVersionKind(schema.GroupVersionKind{
```

Comment:
I was thinking (maybe that is not the right way) to do a kubectl get crd servicemonitor not to search if there is any server monitor object in the cluster :)

Reply:
I think we can try getting the CRD (e.g. [see this](https://github.com/openshift-kni/cnf-features-deploy/pull/1294/files#diff-a64d097cefbbb631fa530e46f04c861be797e765c191ba4491e401e1e26dc1f5R19)).

The drawback is that we have to add the permission (ClusterRole,ClusterRoleBinding,...) to make the operator read that CustomResourceDefinition resource, but it might end up cleaner.

Reply:
I think it should be ok to add `get` for CRD that should not expose the operator to any security issues :)

Reply:
I had to add the permission to the ClusterRole (instead of Role), as the CustomResourceDefinition is not namespaced.
For the same reason, I had to add a non-namespace client to the SriovOperatorConfigReconicler. 
please, take a look